src/components/input-panel/compiled-spec-display/index.tsx

```diff
@@ -4,20 +4,30 @@ import MonacoEditor from 'react-monaco-editor';
 import { connect } from 'react-redux';
 import ConfigEditor from '../../config-editor/index';
 
-import { Mode, SIDEPANE } from '../../../constants';
+import { LAYOUT, Mode, SIDEPANE } from '../../../constants';
 import { State } from '../../../constants/default-state';
 import CompiledSpecDisplayHeader from '../compiled-spec-header';
 
 type Props = ReturnType<typeof mapStateToProps>;
 
 class CompiledSpecDisplay extends React.PureComponent<Props> {
+  public editor;
+  public componentDidUpdate(prevProps) {
+    if (this.props.compiledVegaPaneSize !== prevProps.compiledVegaPaneSize) {
+      if (this.editor) {
+        this.editor.layout();
+      }
```

Comment:
```suggestion
this.editor && this.editor.layout();
```

Reply:
The linter gives an error on using this expression

Reply:
Oh okay